deduplicator/src/bin/main.rs

```diff
@@ -171,9 +182,10 @@ fn main() -> rusqlite::Result<()> {
 
     // --- Cleanup
 
-    if !&params.keep {
+    if !params.keep {
```

Comment:
Do we need to keep the cleanup at the end of the process ?

Reply:
I think it makes sens to consider that the database is temporary and should not hold any resources except during import process. This will also mainly be more convenient for development as it can take quite a bit of disk if you don't think to remove the files.